efficientdet/keras/efficientdet_arch_keras.py

```diff
@@ -81,7 +88,7 @@ def get_config(self):
     }
 
 
-class ResampleFeatureMap(tf.keras.layers.Layer):
```

Comment:
Are tfv1.keras.layers the same as tf.keras.layers? If they are the same, you can just use tf.keras.layers.

Reply:
No, there are some diffrences, not much but for example this layer will not work with tf.keras.layers.Layer as the call is not call(self, inputs, **kwargs).

Reply:
It seems tf1 Layer and tf2 Layer are same class?

Reply:
All i'm sure is that if I change tf.compat.v1 by tf I got problem with matching signature of base method in class Layer

Reply:
Yeah, you could change all tf.compat.v1 to tf if necessary.